docker/entrypoint_foxx.sh

```diff
@@ -51,14 +51,17 @@ then
   # should be replaced with health check at some point
   sleep 5
   "${DATAFED_DEPENDENCIES_INSTALL_PATH}/bin/cmake" --build build --target install
-  
+
+  # Create flag to indicate container has done its job  
   touch "$install_flag"
   chown -R "$UID":"$UID" "/tmp"
 
 else
-  echo "/tmp/.foxx_is_installed has been found skipping reinstall"
+  echo "$install_flag has been found skipping reinstall"
 fi
 
-echo "Sleeping"
-sleep 1000
-
+# Keep container alive for a little bit, the CI pipelines check that the
+# container actually runs. If the container runs to fast the pipeline check
+# might fail because it wasn't able to determine if the container actually
+# ran.
+sleep 60
```

Comment:
Is there a reason we don't just echo a message and sleep indefinitely? Hardcoding sleep is usually a last resort, especially if there is another indicator of success around the container for that CI job.

Reply:
I'm not sure if there is a good way to kill a container in the CI if you were too sleep indefinitely, and if there was it might view killing it as a issue. Definitely something to think about at least.

Reply:
I don't have a good sense for what would be an improvement here. @AronPerez can you explicitly point to what problems might occur with this? The container is meant to be ethemeral as it is part of a setup step. I'm guessing you are proposing something a bit more event driven to kill it.

Reply:
@AronPerez if you can think of anything lets open up an issue and we can explore it there.

Reply:
Here I created an issue. https://github.com/ORNL/DataFed/issues/1067